### PR TITLE
add 'Send Retro Email' link + JS plumbing

### PR DESCRIPTION
**Pull request by lexelby**
I ended up having to do a fair bit of digging to figure this out.  I really love how it ended up!  jQuery is pretty neat.

### CODE REVIEW

File: app/admin/incident.rb [hunk 1]
```diff
@@ -25,7 +31,18 @@
     column :requires_followup
     column :followup_on
     actions do |incident|
-      link_to 'Sync', sync_admin_incident_path(incident), method: :post
+      item 'Sync', sync_admin_incident_path(incident), method: :post, class: 'member_link'
+
+      # This triggers a modal dialog and posts the results back to the
+      # :send_email member action above.
+      item 'Send Retro Email', '#',
+        class: 'lextest member_link',
```
Comment (joshuatobin) [inline app/admin/incident.rb]: Should this be something other than lextest?

**Comment by joshuatobin** [comment]
Nice work on this. I just left one comment.

**Comment by reidmix** [comment]
Woah, very cool

File: app/admin/incident.rb (hunk 1 above)
Comment (lexelby) [inline app/admin/incident.rb]: haha, whoops.

**Comment by lexelby** [comment]
Okay, now it can pre-fill the subject, CC, and body.  CC comes from the EMAIL_CC env var.  I had to get a little funky to pre-fill the values, and I'll potentially have to get even more funky to make the dialog be wider than 300px.  To avoid that I may end up refactoring this to just build the form and call `.dialog()` itself, circumventing `ActiveAdmin.modal_dialog`.

File: app/admin/incident.rb [hunk 2]
```diff
@@ -25,7 +31,21 @@
     column :requires_followup
     column :followup_on
     actions do |incident|
-      link_to 'Sync', sync_admin_incident_path(incident), method: :post
+      item 'Sync', sync_admin_incident_path(incident), method: :post, class: 'member_link'
+
+      # This triggers a modal dialog and posts the results back to the
+      # :send_email member action above.
+      item 'Send Retro Email', '#',
```
Comment (joshuatobin) [inline app/admin/incident.rb]: Awesome.

**Comment by joshuatobin** [comment]
👍

**Pull request merged**